src/Microsoft.DotNet.Build.Tasks/PackageFiles/partialfacades.targets

```diff
@@ -1,5 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <FillPartialFacadeDependsOn>
+      EnsureBuildToolsRuntime
+    </FillPartialFacadeDependsOn>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(IsPartialFacadeAssembly)' == 'true' AND '$(IsReferenceAssembly)' != 'true'">
+    <!-- Tell ResolveMatchingContract to run and resolve contract to project reference -->
+    <ResolveMatchingContract>true</ResolveMatchingContract>
+    <FillPartialFacadeDependsOn>
+      $(FillPartialFacadeDependsOn);ResolveContract
```

Comment:
It's only used when building a normal partial facade.  That's pre-existing and required in order to resolve the contract.  It's showing as a diff because I refactored the duplicated code into the common targets file.